frontend/openchat-client/src/openchat.ts

```diff
@@ -983,14 +979,17 @@ export class OpenChat extends EventTarget {
     }
 
     isPreviewing(chatId: string): boolean {
+        if (this.isReadOnly()) return true;
```

Comment:
this seems wrong. We're _not_ previewing the chat, we just don't have permission to write to it. It seems like it should be handled at the group permissions level.